test_harness.py

```diff
@@ -353,3 +354,31 @@ async def test_project_6(dut):
     # wait some cycles
     await ClockCycles(dut.wb_clk_i, 1000)
 
+@cocotb.test()
+async def test_project_7(dut):
+    # drive a 5 MHz clock on gpio35
+    dut_clk = Clock(dut.io_in[35], 200, units="ns")
+    cocotb.fork(dut_clk.start())
+    await reset(dut)
+
+    project_number = 8
```

Comment:
need to actually send the wishbone request - see other examples

Reply:
Not sure what do you mean by "send the wishbone request". I am not using it, I removed the addresses and everything related to it as suggested above, so... looking at other examples, am I right to assume this is what I should put:

`clock = Clock(dut.wb_clk_i, 10, units="us")`

Reply:
That wasn't correct, thanks Matt explaining, this is the part that was required:

`await wishbone_write(dut, ADDR_PROJECT, project_number)`